Backpack/build.gradle

```diff
@@ -31,6 +31,7 @@ dependencies {
   implementation "com.android.support:support-annotations:$rootProject.ext.supportLibVersion"
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
   implementation "com.android.support:appcompat-v7:$rootProject.ext.supportLibVersion"
+  implementation "com.android.support:cardview-v7:$rootProject.ext.supportLibVersion"
```

Comment:
Should there be documentation on what 3rd party dependencies backpack will bring in for the sake of transparency? I know @k0nserv has mentioned to our squad that there should be no 3rd party dependency from backpack coming in (even though I don't consider these super helpful Android components 3rd party but rather framework components). Should it be somewhere described that we build upon the `cardview-v7` for the `BpkCard`?

Reply:
Will add extra deps wherever possible to readme and doc comments.